src/grisp_i2c.erl

```diff
@@ -26,6 +27,15 @@
 start_link(DriverMod) ->
     gen_server:start_link({local, ?MODULE}, ?MODULE, DriverMod, []).
 
+%% @doc Send messages.
```

Comment:
We should add more information here:

1. Point to the Linux driver spec this is based on
2. Explain how the arguments map to the driver specification
3. Add some shortchanged example